source/connection-method/batch-process-direct/index.html.md.erb

```diff
@@ -17,36 +17,18 @@ Make sure you only select this connection method when your users do not initiate
   </strong>
 </div>
 
-
-<%= partial 'connection-method/partials/connection-method', :locals => { :connection_method => 'BATCH_PROCESS_DIRECT' } %>
-<hr class="govuk-section-break govuk-section-break--visible">
-
-<%= partial 'connection-method/partials/device-id' %>
-<hr class="govuk-section-break govuk-section-break--visible">
-
-<%= partial 'connection-method/partials/user-ids', :locals => { :connection_method => 'BATCH_PROCESS_DIRECT' } %>
-<hr class="govuk-section-break govuk-section-break--visible">
-
-<%= partial 'connection-method/partials/timezone' %>
-<hr class="govuk-section-break govuk-section-break--visible">
-
-<%= partial 'connection-method/partials/gov-client-local-ips', :locals => { :connection_method => 'BATCH_PROCESS_DIRECT' } %>
-<hr class="govuk-section-break govuk-section-break--visible">
-
-<%= partial 'connection-method/partials/gov-client-local-ips-timestamp', :locals => { :connection_method => 'BATCH_PROCESS_DIRECT' } %>
-<hr class="govuk-section-break govuk-section-break--visible">
-
-<%= partial 'connection-method/partials/mac-address' %>
-<hr class="govuk-section-break govuk-section-break--visible">
-
-<%= partial 'connection-method/partials/user-agent', :locals => { :connection_method => 'BATCH_PROCESS_DIRECT' } %>
-<hr class="govuk-section-break govuk-section-break--visible">
-
-<%= partial 'connection-method/partials/vendor-version', :locals => { :connection_method => 'BATCH_PROCESS_DIRECT' }%>
-<hr class="govuk-section-break govuk-section-break--visible">
-
-<%= partial 'connection-method/partials/gov-vendor-license-id' %>
-<hr class="govuk-section-break govuk-section-break--visible">
-
-<%= partial 'connection-method/partials/gov-vendor-product-name' %>
-<hr class="govuk-section-break govuk-section-break--visible">
+### Ed
+
+<ol class="govuk-list cmq-header-list--numbered">
+  <%= partial 'connection-method/partials/gov-client-connection-method', :locals => { :connection_method => 'BATCH_PROCESS_DIRECT' } %>
+  <%= partial 'connection-method/partials/gov-client-device-id' %>
+  <%= partial 'connection-method/partials/gov-client-local-ips', :locals => { :connection_method => 'BATCH_PROCESS_DIRECT' } %>
+  <%= partial 'connection-method/partials/gov-client-local-ips-timestamp', :locals => { :connection_method => 'BATCH_PROCESS_DIRECT' } %>
+  <%= partial 'connection-method/partials/gov-client-mac-addresses' %>
+  <%= partial 'connection-method/partials/gov-client-timezone' %>
+  <%= partial 'connection-method/partials/gov-client-user-agent', :locals => { :connection_method => 'BATCH_PROCESS_DIRECT' } %>
+  <%= partial 'connection-method/partials/gov-client-user-ids', :locals => { :connection_method => 'BATCH_PROCESS_DIRECT' } %>
+  <%= partial 'connection-method/partials/gov-vendor-license-ids' %>
+  <%= partial 'connection-method/partials/gov-vendor-product-name' %>
+  <%= partial 'connection-method/partials/gov-vendor-version', :locals => { :connection_method => 'BATCH_PROCESS_DIRECT' }%>
+</ol>
```

Comment:
I wonder if there is a Ruby way to give the local parameters in one statement for all partials?

Reply:
I've been taking a look at some Ruby docs and can't see a way of doing that unfortunately